server/util_test.go

```diff
@@ -0,0 +1,186 @@
+package server
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/client/flags"
```

Comment:
Please format this file with `goimports` (we have githooks that can do that for you -> https://github.com/cosmos/cosmos-sdk/tree/master/contrib/githooks)